Dockerfile

```diff
@@ -0,0 +1,65 @@
+FROM java:openjdk-8u66-jdk
+MAINTAINER tobilg <fb.tools.github@gmail.com>
+
+# packages
+RUN apt-key adv --keyserver keyserver.ubuntu.com --recv E56151BF && \
+    echo "deb http://repos.mesosphere.io/debian jessie main" | tee /etc/apt/sources.list.d/mesosphere.list && \
+    echo "deb http://dl.bintray.com/sbt/debian /" | tee -a /etc/apt/sources.list.d/sbt.list && \
+    apt-get update && \
+    apt-get install --no-install-recommends -y --force-yes mesos=0.25.0-0.2.70.debian81 \
+    wget \
+    python \
+    make \
+    gcc \
+    build-essential \
+    g++
+
+# Overall ENV vars
+ENV APP_BASE_PATH /app
+ENV SPARK_VERSION 1.5.2
+ENV MESOS_BUILD_VERSION 0.25.0-0.2.70
+
+# Install Node.js 5.x
+ENV NODE_VERSION v5.1.0
+RUN wget --no-check-certificate https://nodejs.org/dist/$NODE_VERSION/node-$NODE_VERSION-linux-x64.tar.gz && \
+    tar -C /usr/local --strip-components 1 -xzf node-$NODE_VERSION-linux-x64.tar.gz && \
+    rm node-$NODE_VERSION-linux-x64.tar.gz
+
+# Spark ENV vars
+ENV SPARK_VERSION_STRING spark-$SPARK_VERSION-bin-hadoop2.6
+ENV SPARK_DOWNLOAD_URL http://d3kbcqa49mib13.cloudfront.net/$SPARK_VERSION_STRING.tgz
```

Comment:
Go to http://spark.apache.org/downloads.html and select "Direct Download" as download type.